static/app/components/events/interfaces/frame/stacktraceLink.tsx

```diff
@@ -192,7 +192,12 @@ export function StacktraceLink({frame, event, line}: StacktraceLinkProps) {
   const {projects} = useProjects();
   const hasStacktraceLinkFeatureFlag =
     organization?.features?.includes('issue-details-stacktrace-link-in-frame') ?? false;
-  const [isQueryEnabled, setIsQueryEnabled] = useState(!hasStacktraceLinkFeatureFlag);
+  const hasGithubSourceLink =
+    event.platform === 'csharp' &&
```

Comment:
Do we need this platform check? I'd think that if we have `frame.sourceLink` that would be enough right?

Reply:
currently the check is redudant because only `csharp` platform has it. And in the future, if we add `sourceLink` on other platforms, we'd like to see the link render. So I'd say best to delete the check